inert.js

```diff
@@ -388,9 +388,13 @@ class InertManager {
     for (let inertElement of inertElements)
       this.setInert(inertElement, true);
 
-    // Comment these two lines out to use programmatic API only
+    // Comment these lines out to use programmatic API only
     this._observer = new MutationObserver(this._watchForInert.bind(this));
-    this._observer.observe(document.body, { attributes: true, subtree: true, childList: true });
+    if (document.readyState === 'loading') {
+      document.addEventListener('DOMContentLoaded', this._observeBody.bind(this));
```

Comment:
This seems broken - if any inert roots are added in between this constructor being called and the `DOMContentLoaded` event being fired, they won't be correctly set up, right?

Perhaps instead we should do this check at the beginning of this method, and delegate the entire constructor body to a separate method which is called either immediately or after load.